core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/LeanBalancer.scala

```diff
@@ -71,7 +71,8 @@ class LeanBalancer(config: WhiskConfig,
     implicit val ec = ExecutionContextFactory.makeCachedThreadPoolExecutionContext()
     val actorSystema: ActorSystem =
```

Comment:
i just noticed this - do you need to create another actor system?!

Reply:
You right, no need. Removed.